examples/system_remark/src/lib.rs

```diff
@@ -63,11 +61,11 @@ mod system_remark {
         use super::*;
         use dotenv::dotenv;
         use hex_literal::hex;
-        use ink_lang as ink;
         // use pink_web3::ethabi::Uint;
         // use std::time::SystemTime;
 
         #[ink::test]
+        #[ignore]
```

Comment:
don't ignore this, it will expose problems

Reply:
It's true it will throw an error. It seems you will solve this problem in another PR, so I suggest you can bring this back later.

Reply:
no problem